Enigmatry.Entry.CodeGeneration.Configuration/Form/Controls/Checkbox/CheckboxFormControlBuilder.cs

```diff
@@ -12,9 +13,26 @@ public CheckboxFormControlBuilder(string propertyName) : base(propertyName)
     {
     }
 
+    /// <summary>
+    /// Set control default value
+    /// </summary>
+    public CheckboxFormControlBuilder WithDefaultValue(bool defaultValue)
+    {
+        _defaultValue = defaultValue ? "true" : "false";
+        return this;
+    }
+
     public override FormControl Build(ComponentInfo componentInfo)
     {
         var checkboxFormControl = new CheckboxFormControl();
         return Build(componentInfo, checkboxFormControl);
     }
-}
\ No newline at end of file
+
+    // This method is deliberately made private so that users cannot set string default values!
+#pragma warning disable IDE0051 // Remove unused private members
+    private new CheckboxFormControlBuilder WithDefaultValue(string defaultValue)
```

Comment:
@msljivic , This is the easies way to hide original implementation :) Any better ideas?

Reply:
I don't like this. We could move default value out of BaseControlBuilder / BaseControl and place it in appropriate input type e.g. string default value for input, boolean for checkbox etc.

The other option is to make API similar to HTMLInputElement, which has both defaultValue: string and defaultChecked: boolean properties.

I would try the first option.